spec/commands/token/send_link_spec.rb

```diff
@@ -36,6 +36,14 @@
       end
     end
 
+    it "stores a valid shortcode" do
+      service = Token::SendLink.new(email: good_email, secret: good_secret, path: "")
+      expect(MembershipMailer).to receive_message_chain(:login_link, :deliver_later).and_return(true)
+      expect(service.call).to be_truthy
+      expect(service.errors).to be_none
+      expect(TemporaryUserToken.where(shortcode: service.shortcode)).to be_present
+    end
+
```

Comment:
Are all the edge and corner cases covered by the existing tests about invalid inputs?

Reply:
I think they are... or at least, this is no worse?